tests/runCoreTests.cmd

```diff
@@ -11,17 +11,6 @@ if NOT %ERRORLEVEL% == 0 (
     goto end
```

Comment:
Do we need this file?

Reply:
I use `runCoreTests.cmd` and `runCoreTests.sh` when I want to run tests from terminal.